R/collect.R

```diff
@@ -152,12 +152,10 @@ collect_predictions.tune_results <- function(x, ..., summarize = FALSE, paramete
   has_coll_col <- coll_col %in% names
 
   if (!has_coll_col) {
-    msg <- paste0(
-      "The `.predictions` column does not exist. ",
-      "Refit with the control argument `save_pred = TRUE` to save predictions."
+    cli::cli_abort(
+      "The {.code .predictions} column does not exist. Please refit with the
```

Comment:
```suggestion
      "The {.field .predictions} column does not exist. Please refit with the
```

following Emil's lead here